### DISCUSSION TITLE
Possible typo in the Open Orca chat format

### DISCUSSION THREAD

**Discussion by player1537** [abetlen/llama-cpp-python#874] [Q&A]
I was looking through the chat formatting code to figure out how function calling works and I found what looks to be a typo in the Open Orca format:

https://github.com/abetlen/llama-cpp-python/blob/53861c9e530c74579c72e0426b0556bc8ad3f58b/llama_cpp/llama_chat_format.py#L287-L296

When I tested this out myself, I found that the system message only includes the string within the parentheses and not any of the other strings that follow, outside the parentheses.

I don't personally use the Open Orca format so it's not really a blocking issue for me, but I thought it'd be good to at least post about it.

**Comment by gardner** [comment]
Pull Request to resolve this issue: https://github.com/abetlen/llama-cpp-python/pull/947
